arbnode/espresso_delayed_message_fetcher.go

```diff
@@ -322,9 +322,10 @@ func (d *DelayedMessageFetcher) getDelayedMessagesInRange(ctx context.Context, b
 		if err != nil {
 			return err
 		}
-		if seqNum > delayedCount+1 {
-			// We need to panic the node here because something has gone seriously wrong
-			log.Crit("Caff node is skipping delayed messages", "seqNum", seqNum, "delayedCount", delayedCount)
```

Comment:
Maybe I am wrong. Doesn't the `seqNum` represent the message index? And `delayedCount` is the index of delayed messages. I think it is a normal case that `seqNum` > `delayedCount` + 1?

Reply:
seqNum should be equal to `delayedCount` it cant be greater than 1